examples/permissioned-token/assets/permissions.py

```diff
@@ -0,0 +1,129 @@
+from pyteal import *
+
+def approval_program():
+    """
+    This smart contract is the permissions or rules smart contract.
+    If configured in controller, then this sc will ensure 2 rules are
+    followed during a token transfer:
+        a) receiver asset balance should be <=100 after transfer
+        b) both [from, to] accounts must be whitelisted
+    Permissions manager manages the contract permission.
+    It can be changed by the asset manager
+    """
+
+    true = Int(1)
+    max_tokens = Bytes("max_tokens") # maximum token transfer rule
+    whitelist_count = Bytes("whitelist_count") # whitelisted accounts counter
+    controller_app_id = Bytes("controller_app_id") # token's controller application index
+
+    on_deployment = Seq([
+	Assert(And(
```

Comment:
```suggestion
        Assert(And(
```

Reply:
BTW, in Python we use only spaces, and we should never use tabs.

Reply:
I see that in this file tabs and spaces are mixed.

Reply:
fixed